feature-libs/product-configurator/rulebased/components/attribute/types/drop-down/configurator-attribute-drop-down.component.ts

```diff
@@ -33,8 +40,40 @@ export class ConfiguratorAttributeDropDownComponent
   attributeDropDownForm = new UntypedFormControl('');
   group: string;
 
+  protected configRouterExtractorService = inject(
+    ConfiguratorRouterExtractorService
+  );
   protected config = inject(Config);
 
+  protected isInitialRenderingOfDomainValues: boolean = true;
+  protected lastAttrSupplement: Configurator.AttributeSupplement | undefined;
+
+  renderDomainValues$: Observable<boolean> = this.isAsyncPricing
```

Comment:
@Is it not better to name this observable `renderedDomanValues$`?

Reply:
it's a boolean  - not the domain values itself

Reply:
I see.
What do you think about `forceRenderingDomainValues`?